packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -296,46 +336,61 @@
   }
 }
 
+:host([layout="horizontal"]),
+:host([layout="horizontal-single"]) {
+  .stepper-item-header {
+    border-color: var(
```

Comment:
same with this one. It doesn't appear that the stepper-item-header border style rules is actually effecting styling. Can you double check?